### ISSUE TITLE
an abnormal region in the generated image

### ISSUE THREAD

**Issue by DanChen001** [junyanz/pytorch-CycleGAN-and-pix2pix#834]
Hi, Dr Zhu, Thank you for sharing the code of this excellent work.

I use your code for unsupervised image-to-image translation.  The generated image looks good on the whole  except for a small abnormal region in it.   Each  generated image contains an abnormal region, however, the other region looks good.

Did you  run into this problem? Could you give me some suggestions, thanks.

**Comment by junyanz** [comment]
It is mode collapse and sometimes it happens. Either retraining the model or using a small learning rate might mitigate the issue.
